Neos.Flow/Classes/Security/Aspect/LoggingAspect.php

```diff
@@ -65,7 +65,7 @@ public function logManagerAuthenticate(JoinPointInterface $joinPoint)
         $authenticationManager = $joinPoint->getProxy();
         $logMessage = 'No account authenticated';
         if ($authenticationManager->getSecurityContext()->getAccount() !== null) {
-            $logMessage = sprintf('Successfully re-authenticated tokens for account "%s"', $authenticationManager->getSecurityContext()->getAccount()->getAccountIdentifier());
+            $logMessage = sprintf('Successfully re-authenticated tokens for account "%s"', (string) $authenticationManager->getSecurityContext()->getAccount()->getAccountIdentifier());
```

Comment:
nitpick: IIRC our CGL don't add a whitespace between casts and the value
```suggestion
            $logMessage = sprintf('Successfully re-authenticated tokens for account "%s"', (string)$authenticationManager->getSecurityContext()->getAccount()->getAccountIdentifier());
```